src/Web/ClientApp/src/app/components/channel/channel.component.html

```diff
@@ -1,20 +1,73 @@
-<div *ngIf="channel" class="card">
-    <header class="card-header">
-        <h1 class="title card-header-title">{{ channel.name }}</h1>
-        <button class="button is-small is-danger" (click)="deleteChannel(id)"><fa-icon class="icon" [icon]="faTrash"></fa-icon><span>Delete</span></button>
-    </header>
-    <div class="card-content">
+<div *ngIf="channel" class="is-flex is-flex-direction-row is-align-items-center is-justify-content-space-between mb-6">
+    <div>
+        <h1 class="is-size-1">{{channel.appSummary.name}}</h1>
+        <a href="https://{{channel.domain}}" target="_blank">{{channel.domain}}</a>
+    </div>
+    <div *ngIf="channel.appSummary.channels.length > 1 && activeTab != tabs.Settings">
```

Comment:
Doesn't it seem a bit odd to be looking at the parent (app) to list out a channel's siblings (channels)? Shouldn't this view center around an appsummary instead of a channel? Looking at this code it does appear we're looking at "application settings".

Doesn't need to be addressed right now, just a passing thought during review.